src/Pages/Withdrawals.razor

```diff
@@ -168,20 +172,35 @@
                             @if (!_isAmountDisabled)
```

Comment:
I don't think we need this. You are doing kind of validation with messages but it is being done with the proper validation tag. Apart from that if we decide to do it like this, we can remove this if and set the visibility of the FieldHelp to follow the `_isAmountDisabled` var.

Reply:
The part of the code was refactored, we need validation if a user increase a number of BTC or input manually.

Reply:
@markettes Please check again.